docs/UserGuide.md

```diff
@@ -123,17 +123,29 @@ Examples:
     email: [johnd@example.com](mailto:johnd@example.com)  
     address: John street, block 123, #01-011
 
-### Locating persons by name: `find`
+### Locating clients by keywords: `find`
 
-This command allows you to find a specific client in your client list by their **name**
+This command allows you to find a specific client in your client list by their **name**, and optionally by other fields too
 
-Format: `find n/NAME`
+Format: `find n/NAME [a/ADDRESSKEYWORD] [e/EMAILKEYWORD] [t/TAGKEYWORD] [k/KINKEYWORD] [d/DESCKEYWORD] [p/PHONEKEYWORD]`
 
-Examples: Lets say your contact list contains the following contacts: **John Doe** **John Lee** **Alex Lee** **Bobby Chan**
+Examples: Lets say your contact list contains the following contacts:
+<img src="images/ExampleClientList.png" />
 
--   `find John` would return `John Doe` as well as `John Lee`
--   `find Alex Lee` would return `Alex Lee`
--   `find Lee` would return `John Lee` as well as `Alex Lee`
+-   `find n/alex` would return `Alex Lee` as well as `Alex Chan`
+-   `find n/alex p/99824412 ` would return `Alex Chan`
+-   `find n/a` would return `Alex Lee` , `Alex Chan`, as well as `Xavier`
+-   `find n/a t/important` would return `Alex Lee` as well as `Xavier`
+
+**Note:** you can search for multiple keywords, for example `find n/name1 n/name2` would search for all clients with names that contain name1 OR name2
+This also applies to the optional keywords, for example `find n/name1 a/address1 a/address2 would search for all clients with names that contain name1 AND
+have an address that contains address1 OR address2, so
+-   `find n/alex n/bobby` would return `Alex Lee`, `Alex Chan`, and `Bobby`
+-   `find n/a p/86151644 p/12355125` would return `Alex Lee` and `Bobby`
+
+<box type="tip" seamless>
```

Comment:
Leave one line after the \<box type="tip" seamless> because when the code is deployed, it would not be formatted correctly.